openmetadata-ui/src/main/resources/ui/src/components/CustomEntityDetail/CustomPropertyTable.tsx

```diff
@@ -89,6 +89,7 @@ export const CustomPropertyTable: FC<CustomPropertyTableProp> = ({
         dataIndex: 'name',
         key: 'name',
         render: (_, record) => getEntityName(record),
+        sorter: (col1, col2) => col1.name.localeCompare(col2.name),
```

Comment:
We are displaying `displayName ?? name` property from data and sorting over name only, I feel it will be confusing for user

We can use something like below
```js
const name1 = getEntityName(col1);
const name2 = getEntityName(col2);
return name1.localeCompare(name2);
```
Also we have internationalisation support in the application, Can you check if this will work well with other langs as well?